apps/web/components/connected/Landspitali/MemorialCard.tsx

```diff
@@ -0,0 +1,363 @@
+  const fundOptions = slice.json?.speedLimitOptions ?? DEFAULT_FUND_OPTIONS
+
+  const methods = useForm<MemorialCard>({
+    mode: 'onChange',
+    defaultValues: {
+      fund: '',
+      inMemoryOf: '',
+      amountISK: '',
+      senderSignature: '',
+      recipientName: '',
+      recipientAddress: '',
+      recipientPostalCode: '',
+      recipientPlace: '',
+      senderName: '',
+      senderNationalId: '',
+      senderAddress: '',
+      senderPostalCode: '',
+      senderPlace: '',
+    },
+  })
+
+  const {
+    handleSubmit,
+    setValue,
+    watch,
+    formState: { errors },
+    control,
+  } = methods
+
+  const [submitted, setSubmitted] = useState(false)
+  const selectedAmount = watch('amountISK')
+
+  const onSubmit = (data: MemorialCard) => {
+    setSubmitted(true)
+    console.log('Submitted data:', data)
+  }
+
+  const handleReset = () => setSubmitted(false)
+
+  if (submitted) {
+    const data = methods.getValues()
+    return (
+      <Stack space={4}>
+        <Text variant="h2">{formatMessage(m.overview.title)}</Text>
+        <Stack space={2}>
+          <Text variant="h3">{formatMessage(m.overview.senderTitle)}</Text>
+          <Text>
+            {formatMessage(m.overview.senderName)} {data.senderName}
+          </Text>
+          <Text>
+            {formatMessage(m.overview.senderNationalId)} {data.senderNationalId}
+          </Text>
+          <Text>
+            {formatMessage(m.overview.senderAddress)} {data.senderAddress}
+          </Text>
+          <Text>
+            {formatMessage(m.overview.senderPostalCode)} {data.senderPostalCode}
+          </Text>
+        </Stack>
+        <Stack space={2}>
+          <Text variant="h3">{formatMessage(m.overview.recipientTitle)}</Text>
+          <Text>
+            {formatMessage(m.overview.recipientName)} {data.recipientName}
+          </Text>
+          <Text>
+            {formatMessage(m.overview.recipientAddress)} {data.recipientAddress}
+          </Text>
+          <Text>
+            {formatMessage(m.overview.recipientPostalCode)}{' '}
+            {data.recipientPostalCode}
+          </Text>
+          <Text>
+            {formatMessage(m.overview.inMemoryOf)} {data.inMemoryOf}
+          </Text>
+        </Stack>
+        <Stack space={2}>
+          <Text variant="h3">
+            {formatMessage(m.overview.senderSignatureTitle)}
+          </Text>
+          <Text>
+            {formatMessage(m.overview.senderSignature)} {data.senderSignature}
+          </Text>
+        </Stack>
+        <Stack space={2}>
+          <Text variant="h3">
+            {formatMessage(m.overview.memorialCardTitle)}
+          </Text>
+          <Text>
+            {formatMessage(m.overview.amountISK)} {data.amountISK} kr.
+          </Text>
+          <Text>
+            {formatMessage(m.overview.fund)} {data.fund}
+          </Text>
+        </Stack>
+        <Box display="flex" justifyContent="spaceBetween">
+          <Button onClick={handleReset}>
```

Comment:
```suggestion
          <Button onClick={handleReset} variant="ghost" preTextIcon="arrowBack">
```

Reply:
I wonder if having an arrow pointing backwards and using another button variant makes it more obvious what the button does